hugr-core/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+//! Extensible, graph-based program representation with first-class support for linear types.
```

Comment:
is this a copy paste of the lib docs? another thing to go out of date potentially?